docs/manual-doc/statement-syntax/data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -0,0 +1,87 @@
+### Create TAG / EDGE Syntax
+
+```
+CREATE {TAG | EDGE} tag_name|edge_name
+    (create_definition, ...)
+    [tag_edge_options]
+  
+create_definition:
+    prop_name data_type
+    
+tag_edge_options:
+    option [, option ...]
+
+option:
+    TTL_DURATION [=] ttl_duration
+    | TTL_COL [=] prop_name
+```
+
+Nebula's graph schema is composed of tags and edges, either of which may have properties. `CREATE TAG` statement defines a tag with the given name. `CREATE EDGE` statement is to define an edge type.
+
+There are several aspects to this syntax, described under the following topics in this section:
+
+* [Tag Name and Edge Type Name](#tag-name-and-edgetype-name)
+
+* [Property Name and Data Type](#property-name-and-data-type)
+
+### Tag Name and EdgeType Name
+
+* **tag_name and edge_name**
+
+    The name of tags and edgeTypes should be **unique** within the space. Once the name is defined, it can not be altered. The rules of tag and edgeType names are the same as those for names of spaces. See [Schema Object Name](../../language-structure/schema-object-names.md)
+
+### Property Name and Data Type
+
+* **prop_name**
+
+    prop_name indicates the name of properties. It must be unique for each tag or edgeType.
```

Comment:
It must be unique in each tag or edgeType. Unfortunately, I found our code do not do the duplication check.